### PR TITLE
Fix test_emcc_print_search_dirs on win32. NFC

### PR DESCRIPTION
**Pull request by sbc100**
Also, print the search dirs for easier debugging.

### CODE REVIEW

File: tests/test_other.py [hunk 1]
```diff
@@ -617,7 +617,11 @@ def test_emcc_print_search_dirs(self):
     output = self.run_process([EMCC, '-print-search-dirs'], stdout=PIPE).stdout
     self.assertContained('programs: =', output)
     self.assertContained('libraries: =', output)
-    self.assertContained(str(shared.Cache.get_lib_dir(absolute=True)), output)
+    libpath = output.split('libraries: =', 1)[1].strip()
+    libpath = libpath.split(os.pathsep)
+    libpath = [Path(p) for p in libpath]
+    print(libpath)
```
Comment (dschuff) [inline tests/test_other.py:623]: remove debug print?

File: tests/test_other.py [hunk 2]
```diff
@@ -617,7 +617,11 @@ def test_emcc_print_search_dirs(self):
...
+    self.assertIn(shared.Cache.get_lib_dir(absolute=True), libpath)
```
Comment (dschuff) [inline tests/test_other.py:624]: so this is now asserting that the lib dir is contained in one of the split components of the path?

Comment (sbc100) [reply tests/test_other.py:624]: Yes, its more precise.. and it parse the paths via the `Path` constructor above so any slashes get normalized automatically (which was the reason for the failure).

File: tests/test_other.py (hunk 1 above)
Comment (sbc100) [reply tests/test_other.py:623]: Our tests do often write stuff to stdout .. so I could do either way.

File: tests/test_other.py (hunk 2 above)
Comment (dschuff) [reply tests/test_other.py:624]: ok, can you say this in the commit description?

**Review by dschuff — APPROVED** [review]

**Pull request merged**